### ISSUE TITLE
`bb balance` doesn't seem to work anymore

### ISSUE THREAD

**Issue by joeljk13** [RocHack/bb#30]
It keeps giving me an 'Unable to log in to Sequoia' error. Maybe Blackboard changed their interface? I haven't really investigated this very thoroughly.

**Comment by clehner** [comment]
Yes, the accounts area is completely changed. `bb balance` bb will have to be rewritten. Authentication to the accounts site goes now through SAML.

Transaction history is still available but is in HTML instead of JSON (this is relevant to [meal-plan-data-grapher](https://github.com/RocHack/meal-plan-data-grapher) and [bbtools](https://github.com/RocHack/bbtools)). There is also now a "board transactions" form that shows usage of unlimited meal plan swipes.
